pkg/get/tools.go

```diff
@@ -1945,5 +1945,34 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 			{{ .Name }}_{{ .VersionNumber }}_{{ $osStr }}_{{ $archStr }}{{ $extStr }}
 			`,
 		})
+
+	tools = append(tools,
+		Tool{
+			Owner:       "sunny0826",
+			Repo:        "kubecm",
+			Name:        "kubecm",
+			Version:     "0.16.2",
+			Description: "Easier management of kubeconfig. ",
+			BinaryTemplate: `
+			{{ $osStr := "" }}
+			{{ $archStr := "" }}
+			{{ $extStr := ".tar.gz" }}
+
+			{{ if HasPrefix .OS "ming" -}}
+			{{ $osStr = "Windows" }}
+			{{- else if eq .OS "linux" -}}
+			{{ $osStr = "Linux" }}
+			{{- else if eq .OS "darwin" -}}
+			{{ $osStr = "Darwin" }}
+			{{- end -}}
+
+			{{- if eq .Arch "x86_64" -}}
+			{{ $archStr = "x86_64" }}
+			{{- else if eq .Arch "aarch64" -}}
+			{{ $archStr = "arm64" }}
+			{{- end -}}
+
+			v{{.Version}}/{{.Name}}_{{.Version}}_{{$osStr}}_{{$archStr}}{{$extStr}}`,
```

Comment:
Removing the `Version` property, you probably need to remove the `v` from the begining (which is a good practice anyways) and for the second `{{.Version}}`, you need to change it to VersionNumber.
Like this:
```suggestion
			{{.Version}}/{{.Name}}_{{.VersionNumber}}_{{$osStr}}_{{$archStr}}{{$extStr}}`,
```

Reply:
> Removing the `Version` property, you probably need to remove the `v` from the begining (which is a good practice anyways) and for the second `{{.Version}}`, you need to change it to VersionNumber. Like this:


modified in new commit [378e192](https://github.com/karthick-kk/arkade/commit/378e19209bbd68a3dd4246ed90a67d4b4b82af40)

Version number changes as recommended makes sense. Made the code changes accordingly. Thank you!